src/elements/shared-styles.html

```diff
@@ -91,21 +91,11 @@
         font-weight: normal;
       }
 
-      h1 {
-        padding: 8px 8px 24px 14px;
-        font-size: 24px;
-        line-height: 30px;
-        font-weight: 500;
-      }
-
       a {
-        color: var(--dark-primary-color);
+        color: var(--primary-gold);
```

Comment:
Amusingly, `--primary-gold` is a dark blue, while `--dark-primary-color` is gold. The gold doesn't have enough color contrast against a white background.

Reply:
this breaks our but ticket button on hover. looking for a different solution now